src/components/atoms/Timeline/hooks.ts

```diff
@@ -119,6 +122,91 @@ const useTimelineInteraction = ({
   };
 };
 
+const DEFAULT_INTERVAL = 10;
```

Comment:
This can go into constants, right?

Reply:
This was fixed by [refactor: move default interval constant](https://github.com/reearth/reearth-web/pull/277/commits/8dc07898743fe1379e772521ece40247bc591914)